ci/setup-rust.sh

```diff
@@ -0,0 +1,14 @@
+#!bash
```

Comment:
`ci` folder name is misleading (I was thinking it was related to github action...)

We should move this to `misc/cibuildwheel_setup_rust.sh`

Reply:
`setup-rust` can be used outside of `cibuildwheel`

Reply:
done